.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Pull Request
+about: Create a pull request to contribute to the TRD
+labels: 
+
+---
+IMPORTANT NOTICE: This PR complies with the [guidelines for contribution to the TRD](https://tezos-reward-distributor-organization.github.io/tezos-reward-distributor/contributers.html) and is qualified for being compensated by the TRD grant if approved by the maintainers.
+
+This PR resolves the issue <issue ID>. The following steps were performed:
+
+* **Bug Analysis**: If the described issue is a bug report, analyze the reasons resulting in this bug. 
+
+* **Solution**: If the described issue is a bug report, describe the proposed solution for the reported bug.
+
+* **Implementation**: Rough description/explanation of the implementation choices.
+
+* **Performed tests**: Describe the performed tests and if further tests are needed.
+
+* **Documentation**: Make sure to document the added changes in a proper way (Readme, help section, documentation, comments in code if needed)
+
+**Work effort**: Give your estimate of the work effort in hours. This might be adjusted or discussed by the other contributors in order to keep a fair rewarding process for the efforts.
```

Comment:
add checkboxes at the end to fill out:
[] I extented the documentation
[] I extended the CLI documentation
[] I changed the README file if necessary
[] I added exetended the CI tests for the corresponding feature
[]..... what else you can think of

Reply:
Can we remove the "I" part from the checklist?
[] Extended TravisCI...
[] Extended Sphinx...
.. etc